### PR TITLE
Check if in debug mode when importing plugins

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

File: tests/test_configurer.py [hunk 1]
```diff
@@ -2,14 +2,6 @@
 import os
 import pytest
 
-@pytest.fixture
-def dispatcher_debug(monkeypatch):
-    monkeypatch.setenv('DISPATCHER_DEBUG_MODE', 'yes')
-
-@pytest.fixture
-def dispatcher_nodebug(monkeypatch):
```
Comment (volodymyrss) [inline tests/test_configurer.py:10]: oh nice!
Can we test that this has the desired effect on the instrument list?

**Review by volodymyrss — CHANGES_REQUESTED** [review]

File: tests/test_server_basic.py [hunk 2]
```diff
@@ -93,6 +93,41 @@ def test_empty_request(dispatcher_live_fixture):
...
+
+    print("content:", c.text)
+
+    jdata=c.json()
+
+    assert c.status_code == 400
+
+     # parameterize this
+    assert jdata['installed_instruments'] == [] or \
+           jdata['installed_instruments'] == ['spi_acs']
```
Comment (volodymyrss) [inline tests/test_server_basic.py:112]: is this one still needed? we might have moved all plugin tests out?

**Review by volodymyrss — CHANGES_REQUESTED** [review]

File: tests/test_server_basic.py (hunk 2 above)
Comment (burnout87) [reply tests/test_server_basic.py:112]: ops I will adjust it

File: cdci_data_analysis/flask_app/app.py [hunk 3]
```diff
@@ -208,14 +208,16 @@ def common_exception_payload():
...
+        'DISPATCHER_DEBUG_MODE', 'no')  # change the default
 
-    if payload['debug_mode'] == "yes":
-        payload['config'] = {
-            'dispatcher-config': remove_nested_keys(app.config['conf'].as_dict(),
-                                                    ['sentry_url', 'logstash_host', 'logstash_port', 'secret_key',
-                                                     'smtp_server_password'])
-        }
+    # TODO why only in debug_mode ?
+    # if payload['debug_mode'] == "yes":
```
Comment (burnout87) [inline cdci_data_analysis/flask_app/app.py:214]: Is ok to remove the nested keys in any case? Before it was happening only in case of debug_mode

**Review by burnout87 — COMMENTED** [review]
one question regarding the usage of the debug_mode

File: cdci_data_analysis/flask_app/app.py (hunk 3 above)
Comment (volodymyrss) [reply cdci_data_analysis/flask_app/app.py:214]: it should be safe

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**